src/components/ec-table-head/ec-table-head.vue

```diff
@@ -152,8 +155,12 @@ export default {
     }
   }
 
-  &__cell-text--has-max-width {
-    @apply tw-truncate;
+  &__cell-text {
+    @apply tw-whitespace-no-wrap;
```

Comment:
I'm not sure why this was added.

Reply:
This is because when the headers are too long in some languages, the text line breaks. That way we avoid it.